requirements.txt

```diff
@@ -0,0 +1,11 @@
+gunicorn==23.0.0
+mysqlclient==2.2.6
+
+#Django
+Django==5.1.4
```

Comment:
Django 5.1 isn't LTS, 5.2 LTS doesn't come out until April. You _may_ want to hold back to Django 4.2 (currently 4.2.18) until later this year unless this isn't going to be released until after April 2025.

Reply:
I feel Releasing in Apr is very optimistic. I feel this is going to live in summer. Thats why I chose Django 5 version. I can change to 4 version if you like. LMK

Reply:
I will keep 5.1 and will upgrade to 5.2 before release. I will create an issue for that.